plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -137,9 +137,12 @@ def update
       end
 
       def ensure_not_updating_match_labels
-        # blue-green deply is allowed to do this, see template_filler.rb + deploy_executor.rb
+        # blue-green deploy is allowed to do this, see template_filler.rb + deploy_executor.rb
         return if @template.dig(:spec, :selector, :matchLabels, :blue_green)
 
+        # allow manual migration when user is aware of the issue and wants to do manual cleanup
+        return if @template.dig(:metadata, :annotations, :"samson/allow_updating_match_labels") == "true"
```

Comment:
do you have somewhere where you're putting all these annotations as constants? would be good to know what samson is looking for in one place.

Reply:
Readme has all the possible annotations :)